uni/lib/json.icn

```diff
@@ -237,12 +240,23 @@ procedure utoj(x,strict,error)
    jerror.get_err()
 end
 
+procedure tojson(x,strict,error)
+   local j;
+   jerror := ErrorHandler(error,strict)
+
+   if j := _utoj(x,jerror,"raw") then
+      return j
+
+   jerror.get_err()
+end
+
+
 #
 # Given a Unicon structure, produce a JSON equivalent if possible.
 #
 # TODO: Check for fails that need error messages
 #
-procedure _utoj(u,jerror)
+procedure _utoj(u,jerror, raw)
```

Comment:
Suggest adding to comment describing the raw parameter. eg.
if raw is absent or null then Unicon descriptor strings will be added to the JSON string. To produce standard JSON without those descriptors, specify a non-null value for raw.